pkg/cloudprovider/aws/launchtemplate.go

```diff
@@ -220,6 +225,39 @@ func sortedKeys(m map[string]string) []string {
 	return keys
 }
 
+func (p *LaunchTemplateProvider) getBottleRocketUserData(ctx context.Context, constraints *v1alpha1.Constraints, additionalLabels map[string]string) (string, error) {
+	var userData bytes.Buffer
```

Comment:
Perhaps cleaner to use standard strings? I'm not sure why we're using a buffer in the other generator.
```
userdata := fmt.Sprintf(`
  cluster-name ....
`)
if labels := functional.UnionStringMaps(additionalLabels, constraints.Labels); len(labels) > 0 {
  userdata += "[settings..."
  for label, value := range labels {
    userdata += ...
  }
}
```